apps/dashboard/src/lib/features/org/utils/types.ts

```diff
@@ -23,3 +24,6 @@ export type CoursesByOrganizationSiteNameSuccess = Extract<
 >;
 
 export type CoursesByOrganizationSiteName = CoursesByOrganizationSiteNameSuccess['data'];
+
+// dashboard analytics types
+export type GetDashStatsResponse = InferResponseType<OrganisationAnalytics> | null;
```

Comment:
The type `OrganisationAnalytics` has to be inferred directly from api response, the same way other requests here are.

and then you should delete `OrganisationAnalytics` from the repo completely.